### ISSUE TITLE
Incorrect accessibility computation of `del`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#795]
When I hover `<del>` element, 

```html
<del>Deleted</del>
```

popup shows below:

```
<del> Computed Accessibility Properties:

No exposed to accessibility tree (hidden element)
```

![image](https://user-images.githubusercontent.com/34136752/209772856-b4f6ffd5-a4fd-41bb-8a4d-cffd32160e78.png)

## Expected behavior

To my understanding, it will be exposed to accessibility tree, and be `deletion` role.

https://www.w3.org/TR/html-aria/#el-del

**Comment by YusukeHirao** [comment]
Resolved in v4
